sci-biology/foldingathome/foldingathome-8.4.5.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+inherit python-any-r1 scons-utils systemd
+
+DESCRIPTION="Distributed computing project for simulating protein dynamics."
+HOMEPAGE="https://foldingathome.org"
+SRC_URI="
+	https://github.com/FoldingAtHome/fah-client-bastet/archive/refs/tags/v$PV.tar.gz
+		-> foldingathome-client-v$PV.tar.gz
+	https://github.com/CauldronDevelopmentLLC/cbang/archive/refs/tags/bastet-v$PV.tar.gz
+		-> foldingathome-cbang-v$PV.tar.gz
+"
+
+S="${WORKDIR}/fah-client-bastet-$PV"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="video_cards_amdgpu"
+
+DEPEND="
+	acct-group/foldingathome
+	acct-group/video
+	acct-user/foldingathome
+	app-arch/bzip2
+	app-arch/lz4
+	dev-db/sqlite:3
+	dev-libs/expat
+	dev-libs/openssl
+	sys-apps/systemd
+	sys-libs/zlib
+	video_cards_amdgpu? ( dev-libs/rocm-opencl-runtime )
+"
+
+RDEPEND="$DEPEND"
+
+src_compile() {
+	export CBANG_HOME="$WORKDIR/cbang-bastet-v$PV"
+	escons -C "$CBANG_HOME"
+	escons
+}
+
+src_install(){
+
+	mv "$S/fah-client" "$S/foldingathome" || die
+	dobin "$S"/foldingathome
+
+	sed -i "s|fah-client|foldingathome|g" "$S"/install/lin/fah-client.service || die
+
+	systemd_newunit "$S"/install/lin/fah-client.service foldingathome.service
+
+	keepdir /var/log/foldingathome
+	keepdir /var/lib/foldingathome
+
+	insinto /etc/foldingathome
+	doins -r "$FILESDIR/config.xml"
+}
+
+pkg_postinst() {
+	chown foldingathome:foldingathome /var/log/foldingathome/ || die
+	chown foldingathome:foldingathome /var/lib/foldingathome/ || die
```

Comment:
We could do keepdir in src_install instead and set the proper perms there

Reply:
How does that play with putting `acct-user/foldingathome` in RDPEND ?

Reply:
Good point, DEPEND is then indeed correct.